routing/localchans/manager.go

```diff
@@ -112,7 +122,7 @@ func (r *Manager) UpdatePolicy(newSchema routing.ChannelPolicy,
 			TimeLockDelta: uint32(edge.TimeLockDelta),
 			MinHTLCOut:    edge.MinHTLC,
 			MaxHTLC:       edge.MaxHTLC,
-			InboundFee:    newSchema.InboundFee,
+			InboundFee:    models.NewInboundFeeFromWire(inboundFee),
```

Comment:
Think we can just do `newSchema.InboundFee.UnwrapOr(models.InboundFee{})` here to avoid the `ExtractRecords` above?